package.json

```diff
@@ -37,6 +37,7 @@
     "react-textarea-autosize": "^8.3.3"
   },
   "devDependencies": {
+    "@uiw/react-md-editor": "^3.17.2",
```

Comment:
I would say this should be a normal dependency instead of devDependency, but I'm not completely sure how Gatsby works regarding these.